outputs.tf

```
@@ -74,6 +74,11 @@ output "node_security_group_id" {
 # IRSA
 ################################################################################
 
+output "oidc_provider" {
+  description = "The OpenID Connect identity provider (issuer URL without leading `https://`)"
+  value       = replace(try(aws_eks_cluster.this[0].identity[0].oidc[0].issuer, ""), "https://", "")
```

```suggestion
  value       = try(replace(aws_eks_cluster.this[0].identity[0].oidc[0].issuer, "https://", ""), "")
```

Since we love `try` so much we should put it as the first function.

---

Agree?

---

fixed in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1870/commits/c5b3a772c14f49fc278b6761575424c9791bdecb and also added to example outputs